xml.go

```diff
@@ -1,59 +1,87 @@
 package vmap
 
 import (
+	"bytes"
+	"encoding/xml"
+
 	"github.com/haxqer/vast"
 )
 
+var (
+	defaultXMLNS = []byte(`xmlns="http://www.iab.net/videosuite/vmap"`)
+	fixedXMLNS   = []byte(`xmlns:vmap="http://www.iab.net/videosuite/vmap"`)
+)
+
 // VMAP is used to express the structure of the ad inventory as a set of timed ad breaks
 // within a publishers video content
 type VMAP struct {
-	Version string    `xml:"version,attr"`
-	AdBreak []AdBreak `xml:"AdBreak"`
+	XMLName xml.Name `xml:"http://www.iab.net/videosuite/vmap vmap:VMAP"`
+	// Version should probably be 1.0
+	Version  string    `xml:"version,attr"`
+	AdBreaks []AdBreak `xml:"AdBreak"`
 }
 
 // AdBreak is point in time where one or more ads are scheduled for delivery
 type AdBreak struct {
+	XMLName        xml.Name       `xml:"vmap:AdBreak"`
 	TimeOffset     *Offset        `xml:"timeOffset,attr"`
 	BreakType      string         `xml:"breakType,attr"`
 	BreakID        string         `xml:"breakId,attr"`
 	RepeatAfter    string         `xml:"repeatAfter,attr"`
-	AdSource       AdSource       `xml:"AdSource"`
-	TrackingEvents TrackingEvents `xml:"TrackingEvents"`
-	Extensions     Extensions     `xml:"Extensions"`
+	AdSource       AdSource       `xml:"vmap:AdSource"`
```

Comment:
why is aded the extra "vamp" tag?

Reply:
its an xml namespace so that the generated XML will pass validation